scripts/generate-namespaced-install-yaml.sh

```diff
@@ -32,11 +32,15 @@ sed -i -e '/^  namespace: federation-system$/d' "${INSTALL_YAML}"
 # Convert rbac from cluster- to namespace-scoped
 sed -i -e 's/ClusterRole/Role/' "${INSTALL_YAML}"
 
-# Add --limited-scope arg to container
-sed -i -e 's/\(\s*- \)\(--install-crds=false\)/\1\2\n\1--limited-scope=true/' "${INSTALL_YAML}"
+# Add args to container
+sed -i -e '/--install-crds=false/a\
+        - --limited-scope=true\
+        - --federation-namespace=$(FEDERATION_NAMESPACE)\
+        - --registry-namespace=$(CLUSTER_REGISTRY_NAMESPACE)' "${INSTALL_YAML}"
 
 # Add namespace env args to container
- sed -i -e '/terminationGracePeriodSeconds/i\        env:\
+ sed -i -e '/terminationGracePeriodSeconds/i\
+        env:\
         - name: FEDERATION_NAMESPACE\
```

Comment:
Why still update env for `FEDERATION_NAMESPACE` and `CLUSTER_REGISTRY_NAMESPACE` to `install-namespaced.yaml`?

Reply:
The goal is setting these environment variables to the namespace the pod is running in (i.e. via the downward api).

Reference: https://kubernetes.io/docs/tasks/inject-data-application/environment-variable-expose-pod-information/#use-pod-fields-as-values-for-environment-variables